### ISSUE TITLE
Cytoscape is not a function with version 2.6.3

### ISSUE THREAD

**Issue by arddor** [cytoscape/cytoscape.js-edgehandles#30]
Hello,

If I try to require and link the modules, as shown here:
```javascript
var cytoscape = require('cytoscape');
var edgehandles = require('cytoscape-edgehandles');
var jquery = require('jquery');

edgehandles( cytoscape, jquery ); // register extension
```
i get Uncaught ReferenceError: cytoscape is not defined. It works when I import them global (I use webpack) however then I receive an Uncaught TypeError: $container.cytoscape is not a function when I try to:
```javascript
cy.edgehandles(edgeConf);
```

I assume the current version broke something.

**Comment by maxkfranz** [comment]
Could you try the commit?  If it works for you, I'll push a release

**Comment by arddor** [comment]
I can confirm that that the commit works. Thank you very much

**Comment by maxkfranz** [comment]
Published as 2.3.1
